www/index.md

```diff
@@ -0,0 +1,26 @@
+# Compensated
+Build products. Get paid.
+
+
+### Expand Your Sales Channels
+You started with a website, and now you have an iOS app.
+And oh yea, there’s that old Gumroad storefront you were experimenting with that has some recurring payments set up.
```

Comment:
I would make a slight adjustment to acknowledge these are only possible cases:

> Maybe you started with a website, and now you have an iOS app. And oh yeah, maybe there's that old Gumroad storefront you were experimenting with that has some recurring payments set up.

Reply:
Thank you! I like that! (Probably will adjust Friday)